atspi-macros/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "atspi-macros"
 version = "0.1.4"
-edition = "2021"
+edition = "2018"
```

Comment:
Why downgrade the edition?

Reply:
Ah I fixed the reason I had to change that in the first place.